src/modules/asab-config/ConfigContainers/ConfigImport.js

```diff
@@ -0,0 +1,169 @@
+import React, { useState, useRef, useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import {
+	CardBody, Row, Col,
+	Button, Input, Label,
+	FormGroup, FormText, InputGroup,
+	InputGroupText, Card
+} from 'reactstrap';
+import {types} from "./actions/actions";
+
+
+const ConfigImport = (props) => {
+
+	const ASABConfigAPI = props.app.axiosCreate('asab_config');
+	const { t } = useTranslation();
+	const [chosenFilename, setChosenFilename] = useState("No file chosen");
+	const [type, setType] = useState("merge");
+	const [errors, setErrors] = useState(false);
+	const inputFileRef = useRef(null)
+	const formRef = useRef(null);
+
+	useEffect(() => {
+		if (props.configImported) {
+			props.getTree();
+			props.app.Store.dispatch({
+				type: types.CONFIG_IMPORTED,
+				config_imported: false
+			});
+		}
+	}, [props.configImported])
+
+	const chooseFile = () => {
```

Comment:
@Aringoaway Woould you please make some short comment above every function you've created with some short description? e.g.

```
// Getting the reference of the chosen file
const chooseFile = () => {...}
```